targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF52840/device/TOOLCHAIN_ARM_STD/nRF52832.sct

```diff
@@ -1,5 +1,11 @@
-LR_IROM1 0x21000 0x00DF000  {
-  ER_IROM1 0x21000 0x00DF000  {
+#! armcc -E
+
+#if !defined(MBED_APP_START)
+  #define MBED_APP_START 0x21000
+#endif
+
+LR_IROM1 MBED_APP_START 0xfffff  {
```

Comment:
~~0xfffff~~ MBED_APP_SIZE

Reply:
I'm sure you should folow this guide: https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/flash/